cmux-tui/bindings/python/tests/test_package_consumer.py

```
@@ -10,6 +11,14 @@
 
 
 PROJECT = Path(__file__).resolve().parents[1]
+VERSION_MATCH = re.search(
+    r'^version = "([^"]+)"$',
+    (PROJECT / "pyproject.toml").read_text(encoding="utf-8"),
+    re.MULTILINE,
+)
+if VERSION_MATCH is None:
+    raise RuntimeError("pyproject.toml has no project version")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)